### DISCUSSION TITLE
Authelia authentication and authorisation server

### DISCUSSION THREAD

**Discussion by easyfit77** [deeztek/Hermes-Secure-Email-Gateway#51] [General]
Ok so my issue is as follows,

I have installed Hermes-seg on both Ubuntu 18.04LTS and 20.04LTS and in both instances I cannot access the admin interface because the Authelia just wont run.

When I check the status of the service I get the following

cu_admin@test-seg:/usr/lib/systemd/system$ sudo systemctl status authelia.service
● authelia.service - Authelia authentication and authorization server
     Loaded: loaded (/lib/systemd/system/authelia.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Tue 2023-03-14 03:09:12 UTC; 51s ago
    Process: 12127 ExecStart=/usr/bin/authelia --config /etc/authelia/configuration.yml (code=exited, status=1/FAILURE)
   Main PID: 12127 (code=exited, status=1/FAILURE)

Mar 14 03:09:12 test-seg systemd[1]: Started Authelia authentication and authorization server.
Mar 14 03:09:12 test-seg authelia[12127]: time="2023-03-14T03:09:12Z" level=error msg="Configuration: option 'jwt_secret' is required"
Mar 14 03:09:12 test-seg authelia[12127]: time="2023-03-14T03:09:12Z" level=error msg="Configuration: session: option 'secret' is required when using the>
Mar 14 03:09:12 test-seg authelia[12127]: time="2023-03-14T03:09:12Z" level=error msg="Configuration: storage: option 'encryption_key' is required"
Mar 14 03:09:12 test-seg authelia[12127]: time="2023-03-14T03:09:12Z" level=fatal msg="Can't continue due to the errors loading the configuration"
Mar 14 03:09:12 test-seg systemd[1]: authelia.service: Main process exited, code=exited, status=1/FAILURE
Mar 14 03:09:12 test-seg systemd[1]: authelia.service: Failed with result 'exit-code'.

Anyone have any idea on how to rectify this, or perhaps know what I may be doing wrong during the install process?

**Comment by deeztek** [comment]
Issue with install script has been confirmed. Working on a fix. Stand by...

**Comment by deeztek** [comment]
New commit https://github.com/deeztek/Hermes-Secure-Email-Gateway/commit/1b8b13510d4197e619c15197847d593f3f30a828 should fix the issue you are experiencing. Let us know if any more issues. Unfortunately, will have to re-run the installation on a fresh install.

Thanks
